gateway/src/api/latest.rs

```diff
@@ -145,8 +146,9 @@ async fn get_projects_list(
         .iter_user_projects_detailed(&name, limit * page, limit)
         .await?
         .map(|project| project::Response {
-            name: project.0.to_string(),
-            state: project.1.into(),
+            id: project.0.to_string().to_uppercase(),
+            name: project.1.to_string(),
+            state: project.2.into(),
```

Comment:
@GugaGongadze can we actually create structs that wrap around the info that gets sent around as a tuple? I am finding it difficult to read it based on tuple indexing. The structs can be defined in `service.rs` I think, but LMK what you think. This isn't necessarily blocking this PR either.

I see some tests are failing too. Can you look into them?

Reply:
nit: I am fine with leaving this as is for this PR, but something similar to `FindProjectPayload` would be needed here as well.

Resolving this.